File: backend/extract.js
const fs = require('fs');
const pdfParse = require('pdf-parse');

async function extract() {
  const file1 = '/Users/had1/Cognify/data/Dec. 8, 2025.pdf';
  const file2 = '/Users/had1/Cognify/data/Dec. 9, 2025.pdf';
  
  const data1 = await pdfParse(fs.readFileSync(file1));
  console.log('=== DEC 8 ===');
  console.log(data1.text);
  
  const data2 = await pdfParse(fs.readFileSync(file2));
  console.log('=== DEC 9 ===');
  console.log(data2.text);
}

extract().catch(console.error);

File: backend/extract.py
import PyPDF2

def extract_text(pdf_path):
    with open(pdf_path, 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ""
        for page in reader.pages:
            text += page.extract_text() + "\n"
        return text

try:
    print("=== DEC 8 ===")
    print(extract_text('/Users/had1/Cognify/data/Dec. 8, 2025.pdf'))
    print("=== DEC 9 ===")
    print(extract_text('/Users/had1/Cognify/data/Dec. 9, 2025.pdf'))
except Exception as e:
    print("Error:", e)
